flipperdevices/flipperzero-good-faps
# Added W25Q512JV support for spi mem manger app.

# What's new

- This adds chip detection/support for Winbond W25Q512JV to SPI Mem Manager

# Verification 
[Datasheet for W25Q512JV](https://www.winbond.com/resource-files/W25Q512JV%20DTR%20RevD%2006292020%20133.pdf)

Steps to verify:

- Compile the .fap binary using ufbt;
- Upload the binary to the apps folder on Flipper;
- Connect W25Q512JV via SPI;
- Device must discover chip correct and define model and vendor as on picture below; read/write/erase actions with memory should work correct.

# Checklist (For Reviewer)
